### PR TITLE
The Firefox Notes feedback page is no longer wrongly opened on clicking the area around the 'notes' menu button

### PR DESCRIPTION
**Pull request by himanish-star**
fixes: #519 

I have fixed this but the only problem is that the size of the button background has been increased, **which can only be seen when we hover over the menu button**. Well according to me all things work fine, however if you don't prefer these changes then I will have to approach this some other way.

Here is a GIF of the changes
simplescreenrecorder-2018-01-05_15 56 58

### CODE REVIEW

**Comment by shikhar-scs** [comment]
Good work @himanish-star :+1:

**Comment by Natim** [comment]
Let's see what @ryanfeeley think about that :)

**Comment by ryanfeeley** [comment]
Good eye @himanish-star! I think the hover was never added, but I did include it in the original design. I try to match the hover style of the formatting buttons (and other toolbar buttons in Firefox).. Rounded corner, centered vertically in space. https://mozilla.invisionapp.com/share/6VBUYHMRB#/235284916_Desktop_Sidebar

**Comment by himanish-star** [comment]
Ok I'm on it

**Comment by himanish-star** [comment]
I have made the changes. This issue was being caused due to overlapping of buttons and thus when we were clicking the side areas of the menu button the feedback page was getting opened. For **illustration purpose** only I have chosen to show those buttons in **blue**.
Here's a GIF

ezgif com-video-to-gif 6

File: src/sidebar/panel.js [hunk 1]
```diff
@@ -71,6 +71,7 @@ ClassicEditor.create(document.querySelector('#editor'), {
       });
 
       enableSync.onclick = () => {
+        giveFeedbackButton.style.display = 'none';
```
Comment (Natim) [inline src/sidebar/panel.js]: Can you put that inside the enableSyncAction function?

Comment (himanish-star) [reply src/sidebar/panel.js]: Sure as you wish

**Comment by himanish-star** [comment]
There is still one problem which I'm trying to fix. I'll update this PR soon after fixing it.

**Comment by Natim** [comment]
Good job @himanish-star thanks !

**Pull request merged**